docs/sources/administration/plugin-management/plugin-integrate.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrate your plugins
+description: Allow plugin frontends to communicate locally with the backends of other installed plugins.
+labels:
+  products:
+    - enterprise
+    - oss
+    - cloud
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - navigation
+  - customize
+  - configuration
+  - grafana.ini
+  - sandbox
+  - frontend
+weight: 350
+---
+
+# Integrate your plugins
```

Comment:
Follow up - i'm not convinced this is an intuitive way for us to describe this, if its only canvas panel that requires this maybe we should move to canvas documentation

Reply:
Again, since its in the grafana server configuration, not sure this can be controlled by grafana cloud users.